elixir/native/extism_nif/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "extism_nif"
+version = "0.0.1-rc.1"
+edition = "2021"
+authors = ["Benjamin Eckel <bhelx@simst.im>"]
+
+[lib]
+name = "extism_nif"
+path = "src/lib.rs"
+crate-type = ["cdylib"]
+
+[dependencies]
+rustler = "0.25.0"
+extism = { version = "0.0.1-alpha", path = "../../../rust" }
```

Comment:
This makes sense for development and testing, but probably not for release. Probably want:

```toml
extism = { version = "0.0.1-rc.3" }
```

Brings up the same convo about which version of the SDK the clients are using I think.

Reply:
Perhaps we could conditionally swap out for development / test / and release. We could use a matrix to test both local and crates version.

Reply:
We could also release possibly with a vendored version of the library.

Reply:
Curious what your thoughts are @nilslice

Reply:
`extism = { version = "0.0.1-rc.3", path = "../../../rust" }` effectively means, "use `version` from crates.io, unless `path` is found". Where `path` is prioritized as the source of the crate when developing locally.

Not sure if that addresses your point above, but happy to dig in more when we can sync up!